SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchSlice.ts

```diff
@@ -46,6 +46,7 @@ export const FetchEventSearches = createAsyncThunk('EventSearchs/FetchEventSearc
     const assetList = (getState() as any).EventSearch.SelectedAssets as SystemCenter.Types.DetailedAsset[];
     const locationList = (getState() as any).EventSearch.SelectedStations as SystemCenter.Types.DetailedLocation[];
     const groupList = (getState() as any).EventSearch.SelectedGroups as OpenXDA.Types.AssetGroup[];
+    const settings = (getState() as any).Settings as Redux.SettingsState;
 
     const filter = {
```

Comment:
I don't think you want to add the `numberofResults` to the `filter` there is no reason. If you are using `filter` to pass it along use the slice instead - that's what the selector is for

Reply:
I'm not sure what you mean. Would this involve combining `params` with information from `SettingsSlice` to `data` in the ajax call?